CMakeLists.txt

```diff
@@ -94,6 +94,10 @@ if (GHEX_ENABLE_ATLAS_BINDINGS)
 endif()
 target_compile_features(ghexlib INTERFACE cxx_std_14)
 
+# Enable CDash dashboard testing integration
+include(CTest)
+
+# Enable adding of tests etc
 enable_testing()
```

Comment:
```suggestion
# Enable adding of tests etc
enable_testing()

# Enable CDash dashboard testing integration
include(CTest)
```

Should it be like this then?

Reply:
@biddisco any feedback?

Reply:
Doesn't really make any difference which way around you put them. Personally I prefer the way you've done it, but tons of other projects do it the other way around (I'm guessing because cut'n'paste is quite common in cmake scripts and nobody really cares).

Reply:
I'll merge the suggestion then and merge